### PR TITLE
[Doc] Fix some typos in intro docs

### PR DESCRIPTION
**Pull request by cgrigis-da**
It looks like many come from a later addition of chapter 8?

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: docs/source/daml/intro/9_Dependencies.rst [hunk 1]
```diff
@@ -81,11 +81,11 @@ Structuring Projects
 
 As you've seen here, identifiers depend on the package as a whole and packages always bring all their dependencies with them. Thus changing anything in a complex dependency graph can have significant repercussions. It is therefore advisable to keep dependency graphs simple, and to separate concerns which are likely to change at different rates into separate packages.
 
-For example, in all our projects in this intro, including this chapter, our scripts are in the same project as our templates. In practice, that means changing a test changes all identifiers, which is not desirable. It's better for maintainability to separate tests from main templates. If we had done that in chapter 7, that would also have saved us from copying the chapter 7
+For example, in all our projects in this intro, including this chapter, our scripts are in the same project as our templates. In practice, that means changing a test changes all identifiers, which is not desirable. It's better for maintainability to separate tests from main templates. If we had done that in chapter 7, that would also have saved us from copying the chapter 7.
```
Comment (carrielaben-da) [inline docs/source/daml/intro/9_Dependencies.rst:84]: Seems like something has gone missing at the end of the last sentence in this paragraph as well. Copying the chapter 7 what?

File: docs/source/daml/intro/9_Dependencies.rst [hunk 2]
```diff
@@ -65,9 +65,9 @@ Dependencies and Data Dependencies
...
 
-For an extension model like this one, ``data-dependencies`` are appropriate so the chapter 8 project includes the chapter 7 that way.
+For an extension model like this one, ``data-dependencies`` are appropriate so the chapter 9 project includes the chapter 7 that way.
```
Comment (carrielaben-da) [inline docs/source/daml/intro/9_Dependencies.rst:70]:
```suggestion
For an extension model like this one,``data-dependencies`` are appropriate, so the chapter 9 project includes the chapter 7 that way.
```

There should be a noun at the end of this sentence. "includes the chapter 7" what?

**Review by carrielaben-da — CHANGES_REQUESTED** [review]

File: docs/source/daml/intro/9_Dependencies.rst (hunk 1 above)
Comment (cgrigis-da) [reply docs/source/daml/intro/9_Dependencies.rst:84]: My understanding is that we mean the chapter 7 itself.
@bame-da, if I am not mistaken you are the original author?

File: docs/source/daml/intro/9_Dependencies.rst (hunk 2 above)
Comment (cgrigis-da) [reply docs/source/daml/intro/9_Dependencies.rst:70]: Same here, my understanding is that "the chapter 7" *is* the subject of the inclusion :)  Perhaps "the chapter 7 source code"?
@bame-da?

File: docs/source/daml/intro/9_Dependencies.rst (hunk 1 above)
Comment (carrielaben-da) [reply docs/source/daml/intro/9_Dependencies.rst:84]: If so, the best phrasing to use in both instances is "chapter 7" (with no 'the').

**Review by carrielaben-da — APPROVED** [review]
Looks good!

**Pull request merged**